usr/src/lib/libzpool/Makefile.com

```diff
@@ -61,6 +61,7 @@ CLEANFILES += $(EXTPICS)
 
 $(LINTLIB) := SRCS=	$(SRCDIR)/$(LINTSRC)
 $(LINTLIB): ../common/zfs.h
```

Comment:
I'm just asking to do it properly - there's no need for 2 dependency lines, ```$(LIBS): ../common/zfs.h``` will take care of both (even for you on dilos).